e2e/utils/create-project-utils.ts

```diff
@@ -169,13 +170,27 @@ export function runCreateWorkspace(
     command += ` ${extraArgs}`;
   }
 
-  const create = execSync(command, {
-    cwd,
-    stdio: isVerbose() ? 'inherit' : 'pipe',
-    env: { CI: 'true', ...process.env },
-    encoding: 'utf-8',
-  });
-  return create ? create.toString() : '';
+  try {
+    const create = execSync(`${command}${isVerbose() ? ' --verbose' : ''}`, {
+      cwd,
+      stdio: 'pipe',
+      env: { CI: 'true', ...process.env },
+      encoding: 'utf-8',
+    });
+
+    if (isVerbose()) {
+      output.log({
+        title: `Command: ${command}`,
+        bodyLines: [create as string],
+        color: 'green',
+      });
+    }
+
+    return create ?? '';
```

Comment:
The `create` will always be string thanks to `utf-8` encoding so this is not needed. We can just `return create`